### PR TITLE
Create marknhenry-cairo.topojson

### LINKED ISSUE #343: Cairo registers on map!
## Adding Cairo to the map

Hi everyone! This issue is to track the addition of Cairo to the map, which can be found at:
- Longitude: 31.244175
- Latitude: 30.045724

### PR DESCRIPTION
**Pull request by marknhenry**
New pin for Cairo @githubteacher closes #343

### CODE REVIEW

**Comment by ajithkottackal** [comment]
Good Job!!!

File: marknhenry-cairo.topojson [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "type": "Feature", "geometry": {"type": "Point", "coordinates": [31.235712, 30.044420]}, "properties": {"username": "marknhenry"} }
```
Comment (Shuaibs) [inline marknhenry-cairo.topojson]: Wow you're doing good.

**Comment by brianamarie** [comment]
@marknhenry It looks like this is being created from a fork of the class repo instead of from a branch. There's a chance you weren't able to join the team directly from the classroom link. Does that sound familiar?

**Comment by hollenberry** [comment]
Hey @marknhenry ! Just a heads up, I'm planning on closing this out by the end of the week if it's still open, to keep the repo tidy.

Feel free to reopen if you don't get a chance to make the updates @brianamarie recommended in time and want to return in the future.

**Pull request closed without merging**